app/scripts/modules/core/src/managed/constraints/constraintRegistry.tsx

```diff
@@ -83,3 +122,14 @@ export const statefulConstraintOptionsByType: { [type: string]: IStatefulConstra
     },
   },
 };
+
+const statelessConstraintOptionsByType: { [type: string]: IStatelessConstraintConfig } = {
+  'depends-on': {
+    iconName: 'mdConstraintDependsOn',
+    shortSummary: {
+      pass: ({ attributes: { environment } }) => `Already deployed to ${environment?.toUpperCase()}`,
```

Comment:
I agree with your top-level comments that the wording here is challenging. I lean more towards

What do you think of "Successfully deployed to previous environment $environment..." instead of "Already deployed to $environment"? I think that would give more of a connotation that a constraint has been satisfied.

Reply:
I like your suggestion Lorin

Reply:
I updated this to `Already deployed to previous environment ${environment?.toUpperCase()}` since folks seem to appreciate the extra detail. I am keeping 'already' for now due to explicitly avoiding any statements like 'successfully deployed' or 'previously deployed' — when paired up with the existing cards about versions being currently/previously deployed on each environment, it becomes highly ambiguous and could be easily mistaken for a statement about the _current status_ of the environment being referenced. While there's likely still room to improve this, 'already' has the benefit of not implying present-time status.